src/styles/core/_breakpoints.scss

```diff
@@ -147,6 +147,7 @@ $breakpoint-attributes: (
   // Center the content using auto left/right margins
   margin-left: auto;
   margin-right: auto;
+  margin-top: 0;
```

Comment:
Is this strictly necessary? This mixin may be used in a lot of places where this kind of change would be difficult to test. I wonder if there might be a better way of doing this in a more scoped place where this style was needed (same for the other margin change below)

Reply:
I did a bit of digging - it seems like the spacing bug this is meant to fix was only happening when the navigator is turned on. (There's extra spacing between the end of hero section and the gray border)
<img width="1257" alt="Screen Shot 2022-03-11 at 2 42 02 AM" src="https://user-images.githubusercontent.com/87735557/157852095-580923dd-9a24-406b-9b93-ddd5b3fe9e32.png">

So I _believe_ it is necessary to fix it in the mixin that specifically applies to the situation where the navigator is open. 
@dobromir-hristov what's your thought on this?